src/main/java/dev/blaauwendraad/masker/json/KeyContainsMasker.java

```diff
@@ -252,26 +221,12 @@ The actual length of the string is the length minus escape characters (which are
      */
     private void maskNumber(MaskingState maskingState, KeyMaskingConfig keyMaskingConfig) {
         // This block deals with numeric values
-        int targetValueStartIndex = maskingState.currentIndex();
+        maskingState.setCurrentValueStartIndex();
```

Comment:
I have to set index of the current value start / end in order to calculate correct offset in `ValueMaskingContext`, if there's any ideas how to make it better, I would be happy to implement those

Reply:
Just for the record: this means that the index will be set to the start index of the current value? If so, than the following name might be slightly more clear:    `setIndexToValueStartIndex`

Reply:
I would thinking of also making it less implementation specific and have something like `maskingState.beforeMaskableValue()` / `maskingState.afterMaskableValue()`

Reply:
Binding the name to the implementation details raises some issues. When we frame it as if we set the index of the current value to be masked, then I would expect that we do it for every value type, including array and object values. But this is not the case currently.
So I agree with Artur on making it less implementation specific. One option would be to name it `registerIndex` / `clearRegisteredIndex`

Reply:
@donavdey we applied your suggestion with `register` / `clear`